exe/cli/cmd/deal.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"github.com/textileio/powergate/deals"
-	"github.com/textileio/powergate/index/ask"
+	askRunner "github.com/textileio/powergate/index/ask/runner"
```

Comment:
Not completely sure about `runner` (re)name... but will try to permeate the same idea to other indexes in the future too.